civis/base.py

```diff
@@ -161,6 +161,10 @@ def __init__(self, poller, poller_args,
         self.poller = poller
         self.poller_args = poller_args
         self.polling_interval = polling_interval
+
+        # TODO explain this. also, maybe rename to _state_override or something
+        self.__state = None
```

Comment:
I think you should rename this as `_state_override` because I was getting confused with how this was related to `self._state`.